tdrs-backend/tdpservice/parsers/schema_defs/ssp/m1.py

```diff
@@ -14,92 +14,244 @@
+            ),
+            validators.if_then_validator(
+                  condition_field='OTHER_TOTAL_REDUCTIONS', condition_function=validators.isLargerThan(0),
+                  result_field='REDUCTIONS_ON_RECEIPTS', result_function=validators.oneOf((1, 2)),
+            ),
+            validators.if_then_validator(
+                  condition_field='OTHER_TOTAL_REDUCTIONS', condition_function=validators.isLargerThan(0),
+                  result_field='OTHER_NON_SANCTION', result_function=validators.oneOf((1, 2)),
+            ),
+            validators.sumIsLarger([
+                  "AMT_FOOD_STAMP_ASSISTANCE",
+                  "AMT_SUB_CC",
+                  "CASH_AMOUNT",
+                  "CC_AMOUNT",
+                  "CC_NBR_MONTHS"
+                  ], 0)
+          ],
           fields=[
               Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
                     required=True, validators=[]),
               Field(item="3", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.dateYearIsLargerThan(1998),
+                        validators.dateMonthIsValid(),
+                    ]),
               Field(item="5", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isAlphaNumeric()
+                    ]),
               Field(item="2", name='COUNTY_FIPS_CODE', type='string', startIndex=19, endIndex=22,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isNumber(),
+                    ]),
               Field(item="4", name='STRATUM', type='string', startIndex=22, endIndex=24,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isInStringRange(0, 99),
+                    ]),
               Field(item="6", name='ZIP_CODE', type='string', startIndex=24, endIndex=29,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isNumber(),
+                    ]),
               Field(item="7", name='DISPOSITION', type='number', startIndex=29, endIndex=30,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.oneOf([1, 2]),
+                    ]),
               Field(item="8", name='NBR_FAMILY_MEMBERS', type='number', startIndex=30, endIndex=32,
-                    required=True, validators=[]),
+                    required=True, validators=[
+                        validators.isInLimits(1, 2),  # ?? doc says 1-2, but t1 has isNumber
```

Comment:
@jtimpe thank you for catching this. it should be `01-99` (same as for T1 record).